### PR TITLE
update_platform (Get Started, Developing, Porting)

### PR DESCRIPTION
**Pull request by omar-srbd**
### Change Description ###

This PR contains the changes of the following pages:

- https://docs.tizen.org/platform/get-started/open-source-project/
- https://docs.tizen.org/platform/get-started/work-flow/
- https://docs.tizen.org/platform/get-started/git-repo-structure/
- https://docs.tizen.org/platform/developing/installing/
- https://docs.tizen.org/platform/developing/setting-up/
- https://docs.tizen.org/platform/developing/cloning/
- https://docs.tizen.org/platform/developing/building/
- https://docs.tizen.org/platform/developing/building-all/
- https://docs.tizen.org/platform/developing/contributing/
- https://docs.tizen.org/platform/developing/creating/
- https://docs.tizen.org/platform/developing/creating-tic/
- https://docs.tizen.org/platform/developing/flashing/
- https://docs.tizen.org/platform/developing/flashing-rpi/
- https://docs.tizen.org/platform/developing/tips/
- https://docs.tizen.org/platform/porting/overview/
- https://docs.tizen.org/platform/porting/kernel/
- https://docs.tizen.org/platform/porting/system/
- https://docs.tizen.org/platform/porting/graphics-and-ui/
- https://docs.tizen.org/platform/porting/recovery/
- https://docs.tizen.org/platform/porting/multimedia/
- https://docs.tizen.org/platform/porting/connectivity/
- https://docs.tizen.org/platform/porting/location/
- https://docs.tizen.org/platform/porting/telephony/
- https://docs.tizen.org/platform/porting/legacy-porting/

### CODE REVIEW

**Review by safir-srbd — COMMENTED** [review]
@omar-srbd Thank you for your fixes for the Platform section of Tizen Docs. I have provided a few suggestions. And for the parts that I could not directly add suggestions to, here are the changes:

File - *docs/platform/developing/building-all.md*

Line 110 - you can control jobs number using `-j` option.
Suggestion - **You** can control jobs number using `-j` option.


File - *docs/platform/developing/contributing.md*

Line 114 - QA engineers are responsible for testing packages as isolated objects, as well as verifying Tizen images with a specific package inside to offer release engineers comprehensive information to make appropriate decision about whether to accept or reject a package, including the following:
Suggestion - QA engineers are responsible for testing packages as isolated objects, as well as verifying Tizen images with a specific package inside to offer release engineers comprehensive information to make appropriate **decisions** about whether to accept or reject a package, including the following:


File - *docs/platform/developing/creating.md*

Line 53 - (added a local repository and only added ***--priority*** option in the end of each line):
Suggestion - (added a local repository and only added ***--priority*** option **at** the end of each line):


File - *docs/platform/developing/flashing-rpi.md*

Line 137 - To create a new flashing profile. For more information see, [edit profile window](#edit-profile-window-fields).
Suggestion - To create a new flashing profile. For more **information,** see [edit profile window](#edit-profile-window-fields).

Line 141 - To modify the existing profile. For more information see, [edit profile window](#edit-profile-window-fields).
Suggestion - To modify the existing profile. For more **information,** see [edit profile window](#edit-profile-window-fields).

Line 153 - This may take few minutes and you may have to enter your supervisor password, which is required to access the SD card.
Suggestion - This may take **a** few **minutes,** and you may have to enter your supervisor password, which is required to access the SD card.

Line 191 - To install RPMs in target device you can specify path to the directory that contains all the required RPMs. You can leave it blank if no RPM is required. You can use this to install Wi-Fi plugins.
Suggestion - To install RPMs in **the** target **device,** you can specify **the** path to the directory that contains all the required RPMs. You can leave it blank if no RPM is required. You can use this to install Wi-Fi plugins.

Line 193 - WiFi
Suggestion - **Wi-Fi**

Line 228 - Select either **Wi-Fi** or **Ethernet** check box to connect to respective network. If not required you can clear both check boxes.
Suggestion - Select either **Wi-Fi** or **Ethernet** check box to connect to respective network. If **it is** not **required,** you can clear both check boxes.

Line 470 - In case of SmartThings devices, the device enables SoftAP mode during setup, therefore, you need not switch to Wi-Fi separately.
Line 470 - In case of SmartThings devices, the device enables SoftAP mode during **setup.** **Therefore,** you need not switch to Wi-Fi separately.


File - *docs/platform/developing/installing.md*

Line 55 - For example in Ubuntu 20.04, append the following line to the source list:
Suggestion - For **example,** in Ubuntu 20.04, append the following line to the source list:

File - *docs/platform/developing/setting-up.md*

Line 95 - The following message indicates that SSH connection has been established successfully:
Suggestion - The following message indicates that **an** SSH connection has been established successfully:

Line 198 - If the Tizen version is lastest, `$Version` equals "".
Suggestion - If the Tizen version is **latest**, `$Version` equals "".


File - *docs/platform/get-started/work-flow.md*

Line 54 - Creating additional branches, such as upstream and development branches to profile projects.
Suggestion - Creating additional branches, such as upstream and development **branches,** to profile projects.


File - *docs/platform/porting/system.md*

Line 2320 - Depending on the architecture, array will contain:
Suggestion - Depending on the architecture, **the** array will contain:

Line 2550 - Resources are consumed by various processes and are classified according to their type: application or service. Applications have the type as native, web app, and .NET, with the state as foreground and background.
Suggestion - Resources are consumed by various processes and are classified according to their type: application or service. Applications have the type as **Native, Web** app, and .NET, with the state as foreground and background.

Line 2865 - The following is example signal observed with `dbus-monitor`:
Suggestion -The following is **an** example signal observed with `dbus-monitor`:

**Review by safir-srbd — APPROVED** [review]
@omar-srbd Thank you for applying the suggested changes.

@asm-foysal @mijin85cho I am approving this PR for merging.

**Review by mijin85cho — COMMENTED** [review]
Hello. About this update I have below questions.
Would you please check?

File: docs/platform/porting/kernel.md [hunk 1]
```diff
@@ -90,45 +90,44 @@ The following figure shows the overview of normal booting sequence in Tizen plat
...
+#### Boot loader setup and build
 
-To build the Tizen TM1 boot loader:
+To build the Tizen TM1 boot loader, follow these steps:
 
 1. Install and set up cross-compile tools on your system if the host has a different architecture than the target (such as x86).
-1. Start with cleaning up the `u-boot-tm1` source. Download the source from the [u-boot-tm1](https://review.tizen.org/git/?p=profile/mobile/platform/kernel/u-boot-tm1.git;a=summary) repository.
+2. Start with cleaning up the `u-boot-tm1` source. Download the source from the [u-boot-tm1](https://review.tizen.org/git/?p=profile/mobile/platform/kernel/u-boot-tm1.git;a=summary) repository:
    ```
    $ make distclean`
```
[the remark this answers is not in the document]
Comment (omar-srbd) [reply docs/platform/porting/kernel.md:110]: Hello @mijin85cho,
_line 125_ of the same file also contains a backtick (`). After a search on the matter, I have noticed that the backtick is normally not used in this manner. 
I am also equally confused on this matter.

[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/platform/porting/kernel.md:110]: I searched the history of this file.

At first, this file was created with [this commit](https://github.com/Samsung/tizen-docs/commit/515ce57d3f2d5f41a2d71e3a0884db59bea401e0#diff-580a78d1191f2b486de923f41aa079f3b5dfea229de9c408ebe84c27ad231d46) and at that time the string was \`$ make distclean\` and \`$ tar cvf bootloader.tar u-boot-mmc.bin\`

Later with [this commit](https://github.com/Samsung/tizen-docs/commit/d773842c3519157e50850c60f171bfd0d31e5c3e#diff-a2e3a9285d076a1522c6c4b00ec9759f9fca17e051c84c2cdd27918e13eac34cR102) the layout was re arranged, changing a command example string(\`) to a command example box(\`\`\`).

As you can see, other \`blah-blah-blah\` were changed to 
\`\`\`
blah-blah-blah
\`\`\`
but these two strings were changed to 
\`\`\`
blah-blah-blah\`
\`\`\`

So I guess it was a mistake..

**Review by safir-srbd — COMMENTED** [review]
@omar-srbd Thank you for making the changes suggested. After another review, I have found a few more issues. The issues which could not be directly suggested are listed below:

*docs/platform/developing/contributing.md*

Line 43 - The patch is merged or discarded depending on the voting results.The merge is performed if:
Suggestion - The patch is merged or discarded depending on the voting results. **The** merge is performed if:

*docs/platform/developing/flashing-rpi.md*

Line 183 & 187 - Downlaod
Suggestion - **Download**

*docs/platform/porting/system.md*

Line 2376 - extension: (string):
Suggestion **extension** (string):

*docs/platform/porting/telephony.md*

Line 5 - The Tizen Telephony features include:
Suggestion - The Tizen Telephony features include **the following**:

Line 55 - With `libtcore`, you can:
Suggestion - ith `libtcore`, you can **do the following**:

**Review by safir-srbd — APPROVED** [review]
@omar-srbd Thank you for incorporating the suggestions. Everything looks good now.

@asm-foysal @mijin85cho I am approving this PR for merging.

**Comment by asm-foysal** [comment]
@mijin85cho Hello!
We updated and reviewed the files in the PR and found everything to be in order. I'm going ahead and merging this PR.

**Pull request merged**